monitoring/onpremise/prometheus/variables.tf

```
@@ -33,6 +33,11 @@ variable "metrics_exporter_url" {
   type        = string
 }
 
+# MongoDB metrics exporter url 
+variable "mongo_metrics_exporter_url" {
+  type = string
```

Add a description, and make it optional
```suggestion
  description = "MongoDB metrics exporter url"
  type        = string
  default     = null
```